framework/src/postprocessors/ExtremeValueBase.C

```diff
@@ -96,8 +86,8 @@ ExtremeValueBase<T>::threadJoin(const UserObject & y)
   const auto & pps = static_cast<const ExtremeValueBase<T> &>(y);
 
   if (((_type == ExtremeType::MAX || _type == ExtremeType::MAX_ABS) &&
-       pps._proxy_value > _proxy_value) ||
-      (_type == ExtremeType::MIN && pps._proxy_value < _proxy_value))
+       pps._proxy_value.first > _proxy_value.first) ||
+      (_type == ExtremeType::MIN && pps._proxy_value.first < _proxy_value.first))
```

Comment:
I think this will be less consistent in parallel in the cases where you have equality in the proxY_value ?

Reply:
Oh! I didn't think about that. I'll revert this change.

Reply:
sounds good. I added the sweep recipes to make sure test suite passes in either configuration